.github/actions/lint-front-matter/rules/url-length.js

```diff
@@ -0,0 +1,23 @@
+const id = 'url-length';
+const Rule = require('./rule');
+const path = require('path');
+const {ok, fail} = require('../utils/status');
+
+/**
+ * @param {string} file Filename of the file being linted.
+ * @return {Object} The result of the check and any failure/warning messages.
+ */
+const test = (file) => {
+  const dirname = path
+    .dirname(file)
+    .split('/')
+    .pop();
+
+  if (dirname.split('-').length === 1) {
```

Comment:
So I'm allowed to have an article as long as it looks like a Custom Element name?

Reply:
yep. The intent is to catch things like `web.dev/javascript` — someone accidentally squatting on a high value path. I can update the comment to clarify that better.